<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tv_AirPortsTable">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>600</width>
      <height>690</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>690</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>6</y>
      <width>369</width>
      <height>250</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>250</height>
     </size>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>26</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Направление и дата</string>
     </property>
     <widget class="QDateEdit" name="dt_dateEnter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>80</width>
        <height>46</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rb_departure">
         <property name="text">
          <string>Вылет</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rb_arrival">
         <property name="text">
          <string>Прибытие</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="pb_request_shedule">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>221</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Получить расписание</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>26</y>
       <width>226</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0">
       <widget class="QComboBox" name="cb_AirportsList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tl_AirportsList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Список Аэропортов</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_statusConnect">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>704</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Отключено</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>447</y>
      <width>408</width>
      <height>250</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>250</height>
     </size>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuaddConnection">
    <property name="title">
     <string>Подключение</string>
    </property>
    <addaction name="act_addData"/>
    <addaction name="separator"/>
    <addaction name="act_connect"/>
   </widget>
   <addaction name="menuaddConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_addData">
   <property name="text">
    <string>Подключить</string>
   </property>
  </action>
  <action name="act_connect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
